lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -291,6 +295,20 @@ def supported_features(cls) -> List[Type[Feature]]:
             features.StartStop,
         ]
 
+    def get_image_info(
```

Comment:
It's not be used out of platform, so it should be a private member.

Reply:
updated